spec/models/split_spec.rb

```diff
@@ -443,7 +443,7 @@
       end
 
       it "overrides to 100% remote_kill_override_to if remote killed" do
-        fc = FactoryBot.create(:app_remote_kill, split: subject, first_bad_version: "0.9", fixed_version: "1.1")
+        fc = FactoryBot.create(:app_remote_kill, split: subject, override_to: "touch_this", first_bad_version: "0.9", fixed_version: "1.1")
```

Comment:
I apparently had some mystery guest testing going on where I was asserting on the factory default. woops. :)